community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_4/notifications/checkForLoadCsvAndMatchOnLargeLabel.scala

```diff
@@ -17,20 +17,20 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.cypher.internal.compatibility.v3_4
+package org.neo4j.cypher.internal.compatibility.v3_4.notifications
 
 import org.neo4j.cypher.internal.frontend.v3_4.notification.{InternalNotification, LargeLabelWithLoadCsvNotification}
 import org.neo4j.cypher.internal.planner.v3_4.spi.PlanContext
-import org.neo4j.cypher.internal.util.v3_4.{Cardinality, LabelId, NameId}
+import org.neo4j.cypher.internal.util.v3_4.{Cardinality, LabelId}
 import org.neo4j.cypher.internal.v3_4.logical.plans.{LoadCSV, LogicalPlan, NodeByLabelScan}
 
-case class CheckForLoadCsvAndMatchOnLargeLabel(planContext: PlanContext,
+case class checkForLoadCsvAndMatchOnLargeLabel(planContext: PlanContext,
```

Comment:
In this case for consistency with the other `NotificationChecker`s.

Reply:
Well, before there were 2 of them, one which started with a lowecase letter and one which started with uppercase. I like that you made it more consistent, but what you did was to change the uppercase to lowercase and create a new one, also lowercase. It would be exactly the same amount of change to make them all uppercase, which I think is more correct.